internal/controllers/provisioningrequest_controller.go

```diff
@@ -2558,3 +2520,46 @@ func (r *ProvisioningRequestReconciler) SetupWithManager(mgr ctrl.Manager) error
 			})).
 		Complete(r)
 }
+
+const (
```

Comment:
These should be taken directly from the governance-policy-propagator repo already imported, as in [here](https://github.com/openshift-kni/oran-o2ims/blob/main/internal/controllers/provisioningrequest_controller.go#L1134).

Reply:
Fixed